src/controls/editor/edithandler.js

```diff
@@ -1235,6 +1300,14 @@ function editAttributes(feat) {
           } else {
             alert('Villkor verkar inte vara rätt formulerat. Villkor formuleras enligt principen change:attribute:value');
           }
+        } else if (obj.type === 'checkboxgroup') {
+          if (obj.options && obj.options.length > 0 && obj.val) {
+            const separator = obj.separator ? obj.separator : ';';
+            obj.val = obj.val.split(separator);
+          }
+          obj.isVisible = true;
+          obj.elId = `input-${obj.name}`;
```

Comment:
This is not related tables proof. Id should be truly unique, e.g. by including layer name like in the else clause:
```obj.elId = `input-${currentLayer}-${obj.name}`;```
Not sure why same code is in several if:s though. maybe my fault.